ui/src/shared/utils/downloadTimeseriesCSV.ts

```diff
@@ -4,21 +4,20 @@ import {unparse} from 'papaparse'
 
 // Utils
 import {timeSeriesToTableGraph} from 'src/utils/timeSeriesTransformers'
-import {executeQuery} from 'src/shared/apis/query'
+import {executeQuery as executeInfluxQLQuery} from 'src/shared/apis/query'
```

Comment:
what do you think about changing the original function names? now that we have these two concepts, these aliases could be useful generally

Reply:
that would be nice, but I'd like to do it in another PR